src/machines/modelingMachine.ts

```diff
@@ -1928,14 +1944,18 @@ export const modelingMachine = setup({
       invoke: {
         src: 'animate-to-face',
         id: 'animate-to-face',
+
         input: ({ event }) => {
           if (event.type !== 'Select default plane') return undefined
           return event.data
         },
+
         onDone: {
           target: 'Sketch',
           actions: 'set new sketch metadata',
         },
+
+        onError: 'Sketch no face',
```

Comment:
Unrelated to "sketch on chamfer"

When working on this PR I created a bug that could get the app into a bad state, before fixing the bug I made sure it wasn't possible to hit this state again with this diff here, but then also fixed my in-progress bug. Not sure if that makes sense but I would add a test for this if I could, but considering I remove the bug that made it possible I can't. It's more like a guard rail in case there's other thing like that lurking.